libbeat/outputs/codec/format/format.go

```diff
@@ -38,9 +37,5 @@ func New(fmt *fmtstr.EventFormatString) *Encoder {
 }
 
 func (e *Encoder) Encode(_ string, event *beat.Event) ([]byte, error) {
-	serializedEvent, err := e.Format.RunBytes(event)
-	if err != nil {
-		logp.Err("Fail to format event (%v): %#v", err, event)
-	}
-	return serializedEvent, err
+	return e.Format.RunBytes(event)
```

Comment:
what will happen when we use `field_format` matcher, is that whenever there is no such fields in the event, it would keep logging this error and the number of such errors can be massive. @urso @exekias